src/panels/lovelace/cards/hui-shopping-list-card.ts

```diff
@@ -250,6 +298,48 @@ class HuiShoppingListCard extends SubscribeMixin(LitElement)
     }
   }
 
+  private async _toggleReorder() {
+    if (!Sortable) {
+      const sortableImport = await import("sortablejs/modular/sortable.core.esm");
+      Sortable = sortableImport.Sortable;
+    }
+    this._reordering = !this._reordering;
+    await this.updateComplete;
+    if (this._reordering) {
+      this._createSortable();
+    } else {
+      this._sortable?.destroy();
+      this._sortable = undefined;
+    }
+  }
+
+  private _createSortable() {
+    this._sortable = new Sortable(this.shadowRoot!.getElementById("sortable"), {
```

Comment:
We should use a Query Decorator here and cache it.

So this would be a new class variable
```ts
@query("#sortable") private _sortableEl?: HTMLElement;
```

Reply:
We can't cache it as it is removed and readded when you switch reorder mode

Reply:
Good catch!